modules/apps/object/object-test/src/testFunctional/tests/ObjectExportImport.testcase

```diff
@@ -543,6 +545,114 @@ definition {
 	}
 
+	@description = "LPS-194547 - Verify it's possible to import custom object definitions in a M:M relationship"
+	@priority = 5
+	test CanImportObjectDefinitionsInMtMRelationship {
+		property custom.properties = "feature.flag.LPS-148856=true";
+
+		task ("Given a custom folder in the model builder table list with two custom object definitions, Object A and Object B") {
+			JSONObjectFolder.addFolder(
+				folderLabel = "Custom Folder 2210",
+				folderName = "CustomFolder2210");
+
+			ObjectAdmin.openObjectAdmin();
+
+			Click.javaScriptClick(
+				folderLabel = "Custom Folder 2210",
+				locator1 = "ObjectAdmin#SIDEBAR_ITEM");
+
+			for (var objectName : list "A,B") {
+				ObjectAdmin.addNewObject(
+					fieldLabelName = "Custom Object ${objectName}",
+					fieldName = "CustomObject${objectName}",
+					pluralLabelName = "Custom Objects ${objectName}");
+
+				CreateObject.saveObject();
+			}
+		}
+
+		task ("and Given a M:M relationship between these objects") {
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObjectA",
+				objectName_2 = "CustomObjectB",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "manyToMany");
+		}
+
+		task ("And Given the object definitions are exported") {
+			for (var objectName : list "A,B") {
+				ObjectAdmin.openObjectAdmin();
+
+				Click.javaScriptClick(
+					folderLabel = "Custom Folder 2210",
+					locator1 = "ObjectAdmin#SIDEBAR_ITEM");
+
+				ObjectExportImport.exportObject(
+					objectLabel = "Custom Object ${objectName}",
+					objectName = "CustomObject${objectName}");
+			}
+		}
+
+		task ("And Given the object definitions are deleted") {
+			for (var objectName : list "A,B") {
+				ObjectAdmin.deleteObjectViaAPI(objectName = "CustomObject${objectName}");
+
+				Refresh();
+			}
+		}
+
+		task ("When the user imports the object definitions") {
+			for (var objectName : list "A,B") {
+				Click(locator1 = "ObjectAdmin#KEBAB_MENU");
+
+				MenuItem.click(menuItem = "Import Object Definition");
+
+				var sampleFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "Object_CustomObject${objectName}*.json");
+				var tempFileDirectory = selenium.getOutputDirName();
+
+				var filePath = "${tempFileDirectory}/${sampleFileName}";
+
+				UploadFile(
+					locator1 = "TextInput#FILE",
+					value1 = ${filePath});
+
+				Type(
+					key_fieldLabel = "Name",
+					locator1 = "TextInput#GENERIC_TEXT_INPUT",
+					value1 = "CustomObject${objectName}");
+
+				Button.clickImport();
+
+				if (IsElementPresent(locator1 = "Button#CONTINUE")) {
+					Click(locator1 = "Button#CONTINUE");
+				}
+			}
+
+			for (var objectName : list "A,B") {
+				ObjectAdmin.moveDefinitionFolder(
+					definitionName = "Custom Object ${objectName}",
+					folderLabel = "Custom Folder 2210");
+
+				Refresh();
+			}
+		}
+
+		task ("Then the object definitions are present and the relationship persists") {
```

Comment:
Here you should use the macro instead the hard content from it @alanafergs. 
e.g. `ObjectAdmin.openObjectFolder(folderLabel = "Custom Folder");`